CodeStarterCamp_Week3/main.swift

```diff
@@ -10,9 +10,11 @@ import Foundation
 
 //Person 타입의 인스턴스로 misterLee , missKim 을 생성
 let missKim = Person(name: "missKim", age: 35, walletInCash: 10000)
-let misterLee = Person(name: "misterLee", age: 31, habit: "농구", personalMBTI: "INFJ", walletInCash: 20000)
+let misterLee = Person(name: "misterLee", age: 31, habit: "농구", MBTI: "INFJ", walletInCash: 20000)
 
 //CoffeeShop 타입의 인스턴스로 yagombucks 을 생성
 var yagombucks = CoffeeShop(owner: "야곰", openingHours: "10:00 ~ 20:00", menu: Coffee.menu, barista: misterLee)
 
-
+missKim.buyCoffee(Coffee.caramelMacchiato, at: yagombucks)
+missKim.walletInCash = 4000
+missKim.buyCoffee(Coffee.caramelMacchiato, at: yagombucks)
```

Comment:
야곰벅스 인스턴스는 굳이 var 로 만들 필요가 없어요!
항상 상수인 `let` 을 기본으로 생각하시고, 에러가 뜨거나 변경될 필요가 있는 경우만 var 로 선언해주세요. 😄 

그리고 메뉴가 3가지나 있고, 돈이 부족한 경우에 `예외 처리`까지 잘 구현해놨기 때문에
그런 것들을 테스트 하기 위해, 미스킴의 주문을 아래와 같이 변경해봤어요!

```suggestion
let yagombucks = CoffeeShop(owner: "야곰", openingHours: "10:00 ~ 20:00", menu: Coffee.menu, barista: misterLee)

missKim.buyCoffee(.caramelMacchiato, at: yagombucks) // 카라멜 마끼아또 시켜보고
missKim.buyCoffee(.americano, at: yagombucks) // 아메리카노도 시켜보고
missKim.buyCoffee(.caffeLatte, at: yagombucks) // 돈 부족한 경우도 테스트해보고
```